libs/design/README.md

```diff
@@ -36,24 +36,22 @@ Refer to the [Upgrade Guide](/libs/design/guides/upgrading.md).
   * [Radio Set](/libs/design/src/atoms/form/radio-set/README.md)
   * [Select](/libs/design/src/atoms/form/select/README.md)
```

Comment:
Should this reference native select instead?

Reply:
yeah, it should. fixed.